src/anemoi/training/diagnostics/mlflow/__init__.py

```diff
@@ -4,3 +4,32 @@
 # In applying this licence, ECMWF does not waive the privileges and immunities
 # granted to it by virtue of its status as an intergovernmental organisation
 # nor does it submit to any jurisdiction.
+
+import os
```

Comment:
are we happing putting this in the __init__ or should it go into a utils.py script for more general functions related to mlflow? No strong opinions in any case!

Reply:
I added a utils.py 😃